manual/expectations.md

```diff
@@ -61,9 +61,14 @@ As your PI, it is my responsibility to do a number of things.
 
 - **Be present:** I expect you to be accessible during normal working hours. 
   Feel free to follow a schedule that works for you, work from home once in a 
-  while when you need to, and take breaks when you want to. But, by default, 
-  plan to come to the office during the daytime with regularity.  I won't track 
-  your hours, but I will worry if I haven't seen your face all week.
+  while when you need to, and take breaks when you need to. But, by default, 
+  plan to come to the office during the daytime with regularity.  
+  I won't track your hours, but I do want to know if you'll be 
+  out-of-communication, and I will worry if I haven't seen your face all week.
+  If you plan to take a vacation, please let me know in advance, particularly 
+  if we have meetings scheduled while you'll be away. Postdocs are considered 
```

Comment:
Graduate students are considered staff as well, right?

Reply:
yes, as below.

Reply:
(this section is about postdocs... the section below is about grad students... it also includes the text about being staff...)